apps/examples/binary_update/Makefile

```diff
@@ -19,16 +19,22 @@
 -include $(TOPDIR)/Make.defs
 include $(APPDIR)/Make.defs
 
+CFLAGS += -D__TizenRT__
+CXXFLAGS += -D__TizenRT__
```

Comment:
Could you let me know why this is necessary? It looks unnecessary and it is not a module which comes from outside opensource.